src/apps/rate_limiter/rate_limiter.lua

```diff
@@ -25,12 +25,15 @@ local PACKET_SIZE = 60
 
 function RateLimiter:new (arg)
    local conf = arg and config.parse_app_arg(arg) or {}
-   assert(conf.rate)
-   assert(conf.bucket_capacity)
+   --- By default, limit to 10 Mbps, just to have a default.
+   conf.rate = conf.rate or (10e6 / 8)
+   -- By default, allow for 255 standard packets in the queue.
+   conf.bucket_capacity = conf.bucket_capacity or (255 * 1500)
    conf.initial_capacity = conf.initial_capacity or conf.bucket_capacity
    local o =
    {
       rate = conf.rate,
+      leaky = conf.leaky,
```

Comment:
I don't like the name “leaky” as it is usually used to describe leaking resources, which is not the case here. I think “drop” is the right name here.

I don't like that the default behavior changes, but on the other hand the fact hat no tests fail make me wonder if `drop=true` is always the wrong choice and if we need it at all. I see two routes here:

1. Show that `drop=false` is consistently superior to `drop=true` for our use cases (hint: the parts of NFV selftest that test `RateLimiter` are currently ridiculously relaxed because the results were... weird). Do not implement `drop` in the first place, instead just change the behavior to `drop=false`.
2. Be conservative and retain the previous default of `drop=true`.